docker-compose.yml

```diff
@@ -25,13 +25,14 @@ x-shared_environment: &shared_environment
   DATABASE_NAME: vapor_database
```

Comment:
We should wrap these so they're not set if using MongoDB

Reply:
This doesn't work at all when testing.
```x-shared_environment: &shared_environment
  LOG_LEVEL: ${LOG_LEVEL:-debug}{{#fluent}}{{^db.is_mongo}}
  DATABASE_URL: mongodb://db:27017/vapor_database{{/db.is_mongo}}{{/fluent}}```
  


At the moment a working solution:
```x-shared_environment: &shared_environment
  LOG_LEVEL: ${LOG_LEVEL:-debug}{{#fluent}}{{^db.is_sqlite}}
  DATABASE_URL: mongodb://db:27017/vapor_database
  DATABASE_HOST: db
  DATABASE_NAME: vapor_database
  DATABASE_USERNAME: vapor_username
  DATABASE_PASSWORD: vapor_password{{/db.is_sqlite}}{{/fluent}}```

Perhaps i'm doing it wrong, please advise.

Reply:
^ 
Means not. Thanks! 

Changes done!